src/yetibot/core/handler.clj

```diff
@@ -52,6 +54,15 @@
        ; ensure prefix is actually a command
        (filter #(command? (-> % second second second)))))
 
+(defn has-command-prefix?
+  "Returns true if body has an command matching the prefix"
```

Comment:
This docstring isn't quite right: this function will return the body of the command if it looks like a command, otherwise nil. The semantics match `re-find`. Because of this I'd probably rename the function (I typically only use `?` suffix if the function actually returns a boolean). Maybe something like `extract-command`.